docs/admin/config/batch_changes.mdx

```diff
@@ -371,16 +396,12 @@ batchSpecLibrary(first: 100) {
 }
 ```
 
-### Featured Templates
-
-<Callout type="note">Featured templates are supported in Sourcegraph v6.4 and more.</Callout>
+### Featured Records
 
-Site-admins can mark a template as featured by either clicking the star button next to the list of library records. Featured records will automatically move to a section atop the remaining library records.
+Site-admins can mark a record as featured by either clicking the star button next to the list of library records. Featured records will automatically move to a section atop the remaining library records. This will add or remove the `"featured"` [label](#labels).
 
 ### Labels
 
-<Callout type="note">Labels are supported in Sourcegraph v6.4 and more.</Callout>
-
 Batch Spec Library records support an optional `labels` field for categorization and filtering. Common labels include:
```

Comment:
Since you say "Common labels include..", I expected to see 2+ label examples here. Since we've only got 1 right now, should we instead change to "The only label currently supported is 'featured'."

Reply:
I'll update the wording 👍